modules/appeals_api/spec/support/shared_examples_for_model_validations.rb

```diff
@@ -7,50 +7,6 @@
   all_claimant_headers = %w[X-VA-Claimant-First-Name X-VA-Claimant-Middle-Initial X-VA-Claimant-Last-Name
                             X-VA-Claimant-Birth-Date].freeze
 
-  describe '#date_formats_are_valid' do
```

Comment:
I feel like we should either keep these tests around, or add a date format checker to [DetailedSchemaErrors ](https://github.com/department-of-veterans-affairs/vets-api/blob/master/spec/lib/common/exceptions/detailed_schema_errors_spec.rb)to ensure that date errors are responding in a way we want. Just removing them like this doesn't specifically test that the change to using the date format works as desired.

Reply:
Went the detailed schema errors route!